src/WebJobs.Script.WebHost/Security/Authorization/FunctionAuthorizationHandler.cs

```diff
@@ -14,11 +14,11 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
         {
             HttpTriggerAttribute httpTrigger = resource.GetTriggerAttributeOrNull<HttpTriggerAttribute>();
 
-            if (httpTrigger != null && PrincipalHasAuthLevelClaim(context.User, httpTrigger.AuthLevel))
+            if (httpTrigger != null && PrincipalHasAuthLevelClaim(context.User, httpTrigger.AuthLevel, null, resource.Metadata.IsDisabled))
```

Comment:
nit: remove null and use named parameter (e.g. `PrincipalHasAuthLevelClaim(context.User, httpTrigger.AuthLevel, isFunctionDisabled: resource.Metadata.IsDisabled)` )